react-search-component/my-app/src/SearchComponent.tsx

```diff
@@ -0,0 +1,57 @@
+import { ChangeEvent, useState } from 'react';
+
+type Props = {
+  quotes: string[];
+};
+
+export function SearchComponent({ quotes }: Props) {
+  const [query, setQuery] = useState('');
+
+  const handleSearch = (query: string) => {
+    setQuery(query);
+  };
+  const matchQuotes = quotes.filter((el: string) =>
+    el.toLowerCase().includes(query.toLowerCase())
+  );
+
+  if (!matchQuotes) {
```

Comment:
`matchQuotes` is an array. It is guaranteed to always be an array, though it may be an empty array.

Arrays are _never_ falsy, even if they are empty. So, `!matchQuotes` will never be truthy. That means this code block will never execute.